src/WordPress/ClientActions.php

```diff
@@ -14,6 +14,7 @@ class ClientActions
     private $dataStore;
     private $logger;
     private $request;
+    private $wordpressClientAPI;
```

Comment:
`$api` is `$wordpressClientAPI` for ClientActions right? I think you can remove `$wordpressClientAPI` and just use `$api`.